metagraph/tests/annotation/row_sparse/test_row_sparse.cpp

```diff
@@ -0,0 +1,100 @@
+#include <cstdint>
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include <graph/representation/succinct/dbg_succinct.hpp>
+
+#include "annotation/binary_matrix/row_sparse/row_sparse.hpp"
+#include "common/utils/file_utils.hpp"
+
+namespace {
+using namespace mtg;
+using namespace testing;
+using ::testing::_;
+
+TEST(RowSparse, Default) {
+    annot::binmat::RowSparse rowdiff;
+    EXPECT_EQ(0, rowdiff.num_relations());
+    EXPECT_EQ(0, rowdiff.num_columns());
+    EXPECT_EQ(0, rowdiff.num_rows());
+}
```

Comment:
We already got nice templated tests for BinaryMatrix and Annotators derived from them. Just add this as another template, and you might need to add a simple convertor.

See for instance, tests/annotation/test_annotation.hpp and tests/annotation/test_matrix.cpp.